### ISSUE TITLE
Spyder / Conda dependency errors on latest build

### ISSUE THREAD

**Issue by loewenm** [ColinDuquesnoy/QDarkStyleSheet#276]
### Describe Your Environment

Attempting to update qdarkstyle to latest version 3.0.2 results in several incompatible packages.

New Spyder version 5.0.0 has released. Spyder will update accordingly via conda, but will not launch due to dependency on qdarkstyle. qdarkstyle will not update to 3.0.2 appropriately.

- QDarkStyle: 2.8.1 (trying to install 3.0.2)
- OS: Windows 10
- Python: 3.8
- Installed using latest conda build (conda update anaconda)

### Language

Python

### Description / Steps to Reproduce [if necessary]

[Description of the issue]

1. [First Step] conda install -c conda-forge qdarkstyle=3.0.2

### Actual Result

Installation error: 

(tensorflow) C:\>conda install -c conda-forge qdarkstyle=3.0.2
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: |
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abor/

spends 30min checking issues...

(see attached file for full read out)
[qdarkstyle.txt](https://github.com/ColinDuquesnoy/QDarkStyleSheet/files/6265173/qdarkstyle.txt)

### Expected Results / Proposed Result

Preparing transaction: done
Verifying transaction: done
Executing transaction: done

### Relevant Code [if necessary]

See above

```
# code here to reproduce the problem

conda install -c conda-forge qdarkstyle=3.0.2

```

**Comment by dpizetta** [comment]
@loewenm hi, I think that not related to the qdarkstyle. I could not reproduce here and by looking at your file, there is no explicit problem related to the qdarkstyle. Our only dependency is QtPy. 

I use Conda and sometimes I found this kind of bug when trying to update things. I'd recommend you create a fresh environment and see how it goes. Let me know the results. Tks

My output:

```
conda install -c conda-forge qdarkstyle=3.0.2
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: D:\miniconda3\envs\test

  added / updated specs:
    - qdarkstyle=3.0.2


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    qdarkstyle-3.0.2           |     pyhd8ed1ab_0         310 KB  conda-forge
    qtpy-1.9.0                 |             py_0          34 KB  conda-forge
    ------------------------------------------------------------
                                           Total:         345 KB

The following NEW packages will be INSTALLED:

  qdarkstyle         conda-forge/noarch::qdarkstyle-3.0.2-pyhd8ed1ab_0
  qtpy               conda-forge/noarch::qtpy-1.9.0-py_0


Proceed ([y]/n)? y


Downloading and Extracting Packages
qdarkstyle-3.0.2     | 310 KB    | ################################################################################################################################################################################################# | 100%
qtpy-1.9.0           | 34 KB     | ################################################################################################################################################################################################# | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
```

**Comment by ccordoba12** [comment]
Hey @loewenm, thanks for reporting. As @dpizetta said your problem is not related to qdarkstyle, but probably to trying to mix the conda-forge and defaults channels.

This problem will be fixed once the Anaconda team releases a new package for qdarkstyle in defaults. Unfortunately, they released Spyder 5 without its correct dependencies, which is causing a lot of headaches to our users (see spyder-ide/spyder#15113).

Until that's fixed, the only solution is to install our 4.2.5 version, as I mentioned in that issue.

**Comment by loewenm** [comment]
Thank you @dpizetta and @ccordoba12 !!!

I'll wait for the new release. :)
